### PR TITLE
Enable option for tailing logs

### PR DESCRIPTION
**Pull request by tlunter**
@bfulton

### CODE REVIEW

**Comment by bfulton** [comment]
Can we default to false?

**Comment by tlunter** [comment]
@bfulton sure, changed.

File: README.md [hunk 1]
```diff
@@ -337,6 +337,8 @@ Options:
 - post_deploy_command - Command run after the deploy has finished
 - sleep_before_termination - Time to wait after the new instance spawns and before terminating the old instance (allow time for external load balancers to start working)
 - ssl - Boolean that enables/disables ssl when doing the live check (default: 'false').
+- log_files - Paths to files on each instance to tail during deploy (default: '/var/log/syslog')
+- tail_logs - Boolean that will enable/disable tailing log files (default: 'true')
```
Comment (bfulton) [inline README.md]: Need to change here too.

Comment (tlunter) [inline README.md]: good call, updated.

**Comment by bfulton** [comment]
+1

**Pull request merged**